src/endpoint/sts/sts_rest.js

```diff
@@ -90,6 +90,8 @@ async function handle_request(req, res) {
     }
     req.op_name = op_name;
 
+    http_utils.validate_bucket_name_object_key_length(req);
```

Comment:
I don't think this is relevant to sts/iam requests at all - we don't parse a bucket/key params in those servers.